config/routes.rb

```diff
@@ -35,6 +35,8 @@
   get '/about', to: 'about#show'
   get '/get_started', to: 'get_started#show'
 
+  post '/friendships/:user_id/:friend_id', to: "friendships#create"
```

Comment:
route to create a new `friendship`, considering using hidden params with the `Add as Friend` button to make the path simpler